homeassistant/components/rfxtrx/cover.py

```diff
@@ -95,20 +125,48 @@ async def async_added_to_hass(self):
             if old_state is not None:
                 self._state = old_state.state == STATE_OPEN
 
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        supported_features = SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_STOP
+
+        if self._venetian_blind_mode in (
+            CONST_VENETIAN_BLIND_MODE_US,
+            CONST_VENETIAN_BLIND_MODE_EU,
+        ):
+            supported_features |= (
+                SUPPORT_OPEN_TILT
+                | SUPPORT_CLOSE_TILT
+                | SUPPORT_STOP_TILT
+                | SUPPORT_SET_TILT_POSITION
```

Comment:
We don't seem to support setting tilt position.

Reply:
No, we don't.  Bit it seems it is the only way to enable the cover UI which has the tilt controls.  If there is a better way to achieve that without  SUPPORT_SET_TILT_POSITION (and SUPPORT_STOP_TILT for that matter), some help on how to do it will be greatly appreciated.

Reply:
Then that seem to be a bug that should be fixed in gui.

Reply:
So what are the next steps on this PR?

Reply:
Please remove this line.

Reply:
Sure.  How do I set a dependency on resolving the bug mentioned above?  Releasing this without the bug fixed (or the line in question left) will technically not add any functionality.

Reply:
I would suggest you open a pull on the frontend repo fixing the issue. It aught to mostly be adjusting the value on supported that it checks for.

Reply:
Ps. It's not useless, since even without GUI support, it's possible to call services directly.

Reply:
Line removed